kakao/src/main/kotlin/com/agoda/kakao/picker/date/KDatePicker.kt

```diff
@@ -0,0 +1,49 @@
+package com.agoda.kakao.picker.date
+
+import android.view.View
+import android.widget.DatePicker
+import androidx.test.espresso.DataInteraction
+import com.agoda.kakao.common.builders.ViewBuilder
+import com.agoda.kakao.common.views.KBaseView
+import com.agoda.kakao.text.KButton
+import org.hamcrest.Matcher
+
+/**
+ * View for interact with default date picker
+ *
+ * @see DatePicker
+ */
+class KDatePicker : KBaseView<KDatePicker>, DatePickerAction, DatePickerAssertion {
+    val okButton: KButton
+    val cancelButton: KButton
+
+    constructor(function: ViewBuilder.() -> Unit) : super(function) {
+        okButton = KButton {
```

Comment:
is `okButton` and `cancelButton` descendants of the `DatePicker` widget? If so, please add `isDescendantOfA(function)` to the matcher.